### PR TITLE
T35091 Add method to validate request parameters

### PR DESCRIPTION
**Pull request by JenySadadia**
api/models.py: add 'validate_params' method
api/main.py: validate params in '/nodes' endpoint

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -113,3 +113,19 @@ def modify_parent(cls, query_param_dict):
         """Modify query parameter dictionary for parent"""
         query_param_dict['parent'] = ObjectId(query_param_dict['parent'])
         return query_param_dict
+
+    @classmethod
+    def validate_params(cls, params: dict):
+        """Validate Node parameters"""
+        status = params.get('status')
+        if status:
+            if status not in list(status.value for status in StatusValues):
```
Comment (mgalka) [inline api/models.py:122]: maybe we could use `and` operator instead of a nested `if`?

**Review by mgalka — COMMENTED** [review]
Looks good, one minor issue that may be worth fixing.

File: api/models.py (hunk 1 above)
Comment (JenySadadia) [reply api/models.py]: Yes, also replaced `list( )` with `[ ]` as it affects the execution speed.

**Review by mgalka — APPROVED** [review]
LGTM

**Pull request merged**